src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -393,6 +394,50 @@ public function insert(Table $table, $row)
     /**
      * {@inheritdoc}
      */
+    public function update(Table $table, $row, $whereColumns)
+    {
+        $this->startCommandTimer();
+        $this->writeCommand('update', array($table->getName()));
+
+        if (is_string($whereColumns)) {
+            $whereColumns = [$whereColumns];
+        }
+        if (!is_array($whereColumns)) {
+            throw new InvalidArgumentException;
+        }
+
+        $whereValues = [];
+        $whereParams = [];
+        foreach ($whereColumns as $column) {
+            // Store value for where clause
+            $whereValues[$column] = $row[$column];
+            // Set param name and quote column
+            $whereParams[] = $this->quoteColumnName($column) . "=?";
+            // Remove where values as they shouldn't be updated
+            unset($row[$column]);
```

Comment:
```sql
UPDATE table SET key = value1 WHERE key = value2;
```

The where clause has no bearing on the columns being updated. The unset() would remove this capability.